datacommons/core.py

```diff
@@ -0,0 +1,130 @@
+    dcid: The node to get property labels for.
+    outgoing: Whether or not the node is a subject or object.
+    reload: Whether or not to send the query to cache.
+  """
+  # Generate the GetProperty query and send the request
+  params = "?dcid={}".format(dcid)
+  if reload:
+    params += "&reload=true"
+  url = utils._API_ROOT + utils._API_ENDPOINTS['get_property'] + params
+  res = requests.get(url)
+  payload = utils.format_response(res)
+
+  # Return the results based on the orientation
+  if outgoing and 'outArcs' in payload:
+    return payload['outArcs']
+  elif not outgoing and 'inArcs' in payload:
+    return payload['inArcs']
+  return []
+
+def get_property_values(dcids, prop, outgoing=True, value_type=None, reload=False, limit=utils._MAX_LIMIT):
+  """ Returns values associated to given dcids via the given property.
+
+  If the dcids field is a list, then the return value is a dictionary mapping
+  dcid to the list of values associated with the given property.
+
+  If the dcids field is a Pandas Series, then the return value is a Series where
+  the i-th cell is the list of values associated with the given property for the
+  i-th dcid.
+
+  Args:
+    dcids: A string, list of, or Pandas DataSeries of dcid.
+    prop: The property to get the property values for.
+    outgoing: Whether or not the dcids are subjects or objects.
+    value_type: Filter returning values by a given type.
+    reload: Whether or not to send the query to cache.
+    limit: The maximum number of values to return.
+  """
+  # Convert the dcids field and format the request to GetPropertyValue
+  dcids, req_dcids = utils.convert_dcids_type(dcids)
+  req_json = {
+    'dcids': req_dcids,
+    'property': prop,
+    'outgoing': outgoing,
+    'reload': reload,
+    'limit': limit
+  }
+  if value_type:
+    req_json['value_type'] = value_type
+
+  # Send the request
+  url = utils._API_ROOT + utils._API_ENDPOINTS['get_property_value']
+  res = requests.post(url, json=req_json)
+  payload = utils.format_response(res)
+
+  # Create the result format for when dcids is provided as a list.
+  result = defaultdict(list)
+  for dcid in dcids:
+    if dcid in payload and prop in payload[dcid]:
+      for node in payload[dcid][prop]:
+        if 'dcid' in node:
+          result[dcid].append(node['dcid'])
+        elif 'value' in node:
+          result[dcid].append(node['value'])
+    else:
+      result[dcid] = []
+
+  # Format the result as a Series if a Pandas Series is provided.
+  if isinstance(dcids, pd.Series):
+    return pd.Series([result[dcid] for dcid in dcids])
+  return dict(result)
+
+def get_triples(dcid, reload=False, limit=utils._MAX_LIMIT):
+  """ Returns a list of triples where the dcid is either a subject or object.
+
+  The return value is a list of tuples (s, p, o) where s denotes the subject
+  entity, p the property, and o the object.
+
+  Args:
+    dcid: The node to get triples for.
+    reload: Whether or not to send the query to cache.
```

Comment:
This is confusing. Reading this feels like "reload=true" sends query to cache, which is not true.

Reply:
Agreed, fixed to say "A flag that sends the query without hitting cache when set."

Reply:
Removed reload from the API